src/main/java/org/reservationapplication/Main.java

```diff
@@ -0,0 +1,24 @@
+package org.reservationapplication;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class Main {
+    public static void main(String[] args) {
+
+        List<CoworkingSpace> generalCoworkingSpace = new ArrayList<>();
+        CoworkingSpaceService coworkingSpaceService = new CoworkingSpaceService(generalCoworkingSpace);
+
+        List<Reservation> personalReservation = new ArrayList<>();
+        List<Reservation> generalReservationList = new ArrayList<>();
+        ReservationService reservationService = new ReservationService(personalReservation, generalReservationList);
```

Comment:
Don't forget to do it a repo next time. If I see you reading from file directly here, i'll become violent